bindings/pydeck/pydeck/bindings/layer.py

```diff
@@ -30,10 +30,12 @@ def __init__(
 
         type : str
             Type of layer to render, e.g., `HexagonLayer`
-        id : str
+        id : str, default None
             Unique name for layer
         data : str or list of dict of {str: Any} or pandas.DataFrame
             Either a URL of data to load in or an array of data
+        binary_transport : bool, default None
```

Comment:
what is the convention here.  I like **is_binary_transport**

Reply:
This is a better name